La locale fr_FR.UTF8 est spécifiée explicitement

`make serve` échoue chez moi avec : 

```
15:06 $ make serve
	🍋  MakeCitron 1.4.16		<serve>@profiterole
python3.8 /home/gilles/workspace/www-pellov/pellov.py
Traceback (most recent call last):
  File "/home/gilles/workspace/www-pellov/pellov.py", line 16, in <module>
    setlocale(LC_ALL, 'fr_FR')
  File "/home/gilles/.pyenv/versions/3.8.15/lib/python3.8/locale.py", line 608, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting
make: *** [Makefile:20: run] Error 1
```

Il faudrait appliquer ce diff:

```diff
diff --git a/pellov.py b/pellov.py
index 39bbdd5..1331b3f 100755
--- a/pellov.py
+++ b/pellov.py
@@ -1,7 +1,6 @@
 #!/usr/bin/env python3
 
-from locale import LC_ALL, setlocale
-
+import locale
 import jinja2
 import mandrill
 from flask import (Flask, Response, abort, flash, redirect, render_template,
@@ -13,7 +12,7 @@ app = Flask(__name__)
 app.secret_key = 'secret key'
 app.config.from_envvar('WWWPELLOV_CONFIG', silent=True)
 
-setlocale(LC_ALL, 'fr_FR')
+locale.setlocale(locale.LC_ALL, ("fr_FR", "UTF-8"))
 
 MANDRILL_KEY = app.config.get('MANDRILL_KEY')
 

```

Le pre-commit hook impose de reformater le py, en ensuite c'est le check isort qui échoue.

```
    🞋  lint-python ➘➘
pytest --flake8 --isort -m "flake8 or isort" "/home/gilles/workspace/www-pellov" --ignore=lib/frontend/static
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.8.15, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/gilles/workspace/www-pellov
plugins: flake8-1.0.7, isort-2.0.0
collected 2 items                                                                                                                                                                                                 

pellov.py F.                                                                                                                                                                                                [100%]

==================================================================================================== FAILURES =====================================================================================================
___________________________________________________________________________________________________ isort-check ___________________________________________________________________________________________________
 

 import jinja2
 import mandrill
-from flask import (
-    Flask,
-    Response,
-    abort,
-    flash,
-    redirect,
-    render_template,
-    request,
-    url_for,
-)
+from flask import (Flask, Response, abort, flash, redirect, render_template,
+                   request, url_for)
 
 app = Flask(__name__)
 
============================================================================================= short test summary info =============================================================================================
FAILED pellov.py::ISORT
=========================================================================================== 1 failed, 1 passed in 0.12s ===========================================================================================
make[2]: *** [base.Makefile:313: lint-python] Error 1
make[2]: Leaving directory '/home/gilles/workspace/www-pellov'
make[1]: *** [base.Makefile:321: lint] Error 2
make[1]: Leaving directory '/home/gilles/workspace/www-pellov'
make: *** [base.Makefile:138: pre-commit] Error 2

```

--- a/pellov.py
+++ b/pellov.py
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-from locale import LC_ALL, setlocale
+import locale
 
 import jinja2
 import mandrill
@@ -21,7 +21,7 @@ app = Flask(__name__)
 app.secret_key = "secret key"
 app.config.from_envvar("WWWPELLOV_CONFIG", silent=True)
 
-setlocale(LC_ALL, "fr_FR")
+locale.setlocale(locale.LC_ALL, ("fr_FR", "UTF-8"))
 
 MANDRILL_KEY = app.config.get("MANDRILL_KEY")
 

--- a/pellov.py
+++ b/pellov.py
@@ -4,46 +4,58 @@ from locale import LC_ALL, setlocale
 
 import jinja2
 import mandrill
-from flask import (Flask, Response, abort, flash, redirect, render_template,
-                   request, url_for)
+from flask import (
+    Flask,
+    Response,
+    abort,
+    flash,
+    redirect,
+    render_template,
+    request,
+    url_for,
+)
 
 app = Flask(__name__)
 
 # Secret key doesn't have to be changed, it is only used for flash messages
-app.secret_key = 'secret key'
-app.config.from_envvar('WWWPELLOV_CONFIG', silent=True)
+app.secret_key = "secret key"
+app.config.from_envvar("WWWPELLOV_CONFIG", silent=True)
 
-setlocale(LC_ALL, 'fr_FR')
+setlocale(LC_ALL, "fr_FR")
 
-MANDRILL_KEY = app.config.get('MANDRILL_KEY')
+MANDRILL_KEY = app.config.get("MANDRILL_KEY")
 
 
-@app.route('/')
-@app.route('/<page>')
-def page(page='index'):
+@app.route("/")
+@app.route("/<page>")
+def page(page="index"):
     try:
-        return render_template('{}.html'.format(page), page=page)
+        return render_template("{}.html".format(page), page=page)
     except jinja2.exceptions.TemplateNotFound:
         abort(404)
 
 
-@app.route('/robots.txt')
+@app.route("/robots.txt")
 def robots():
-    return Response('User-agent: *\nDisallow: \n', mimetype='text/plain')
+    return Response("User-agent: *\nDisallow: \n", mimetype="text/plain")
 
 
-@app.route('/contact', methods=['POST'])
+@app.route("/contact", methods=["POST"])
 def contact():
     message = {
-        'to': [{'email': 'contact@kozea.fr'}],
-        'subject': 'Prise de contact sur le site de PromoMaker',
-        'from_email': 'contact@kozea.fr',
-        'html': '<br>'.join([
-            'Prénom : %s' % request.form.get('firstname', ''),
-            'Nom : %s' % request.form.get('lastname', ''),
-            'Email : %s' % request.form.get('email', ''),
-            'Société : %s' % request.form.get('company', ''),
-            'Téléphone : %s' % request.form.get('phone', '')])}
+        "to": [{"email": "contact@kozea.fr"}],
+        "subject": "Prise de contact sur le site de PromoMaker",
+        "from_email": "contact@kozea.fr",
+        "html": "<br>".join(
+            [
+                "Prénom : %s" % request.form.get("firstname", ""),
+                "Nom : %s" % request.form.get("lastname", ""),
+                "Email : %s" % request.form.get("email", ""),
+                "Société : %s" % request.form.get("company", ""),
+                "Téléphone : %s" % request.form.get("phone", ""),
+            ]
+        ),
+    }
 
     if app.debug:
         print(message)
@@ -51,10 +63,11 @@ def contact():
         mandrill.Mandrill(MANDRILL_KEY).messages.send(message=message)
 
     flash(
-        'Nous vous remercions pour votre demande. '
-        'Notre équipe va revenir vers vous dans les plus brefs délais.')
-    return redirect(url_for('page'))
+        "Nous vous remercions pour votre demande. "
+        "Notre équipe va revenir vers vous dans les plus brefs délais."
+    )
+    return redirect(url_for("page"))
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     app.run(debug=True)

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,0 +1,7 @@
+[tool.black]
+line-length = 80
+
+[tool.isort]
+line_length = 80
+profile = "black"
+combine_as_imports = true

